src/Explorer/Controls/NotebookGallery/GalleryViewerComponent.tsx

```diff
@@ -227,7 +227,7 @@ export class GalleryViewerComponent extends React.Component<GalleryViewerCompone
           <Stack.Item styles={{ root: { minWidth: 200 } }}>
             <Dropdown options={this.sortingOptions} selectedKey={this.state.sortBy} onChange={this.onDropdownChange} />
           </Stack.Item>
-          {this.props.container?.isGalleryPublishEnabled() && (
+          {(!this.props.container || this.props.container.isGalleryPublishEnabled()) && (
```

Comment:
>!this.props.container [](start = 12, length = 21)

?
did you mean to do the valid check ? if so, why not just check for container && gallerypublishenabled?

Reply:
`isGalleryPublishEnabled` is checking the feature flag which only works for Data Explorer. However this component is used by standalone gallery too where we don't support feature flags. So to show `InfoComponent` I added another check (in standalone gallery `this.props.container` is always undefined).